metronome/config/specs/src/io/iohk/metronome/config/ConfigParserSpec.scala

```diff
@@ -22,6 +22,24 @@ class ConfigParserSpec
     json.noSpaces shouldBe """{"nested-structure":{"bar_baz":{"spam":"eggs"},"foo":10}}"""
   }
 
+  "toJsonEscape" should "parse and escape escape.conf to JSON" in {
```

Comment:
```suggestion
  it should "deal with escape characters according to rfc7159 (section-7)" in {
```

Reply:
I would actually move this further down to appear under the `withEnvVarOverrides` section.

Reply:
Sure, now it is done